sound/soc/sof/ipc4-topology.c

```diff
@@ -2128,6 +2170,10 @@ static const struct sof_ipc_tplg_widget_ops tplg_ipc4_widget_ops[SND_SOC_DAPM_TY
 				  dai_token_list, ARRAY_SIZE(dai_token_list), NULL,
 				  sof_ipc4_prepare_copier_module,
 				  sof_ipc4_unprepare_copier_module},
+	[snd_soc_dapm_buffer] = {sof_ipc4_widget_setup_pcm, sof_ipc4_widget_free_comp_pcm,
+				 dai_token_list, ARRAY_SIZE(dai_token_list), NULL,
```

Comment:
why are we using a dai_token_list for something that is not a DAI?

This sounds bad to me, I would guess some tokens apply only for real DAIs and not for internal buffer handling?

Reply:
> > why are we using a dai_token_list for something that is not a DAI?
> > This sounds bad to me, I would guess some tokens apply only for real DAIs and not for internal buffer handling?
> 
> Thanks @plbossart change it to host_token_list now.

And rename it to common_copier_token_list after chatting with Ranjani.